src/main/java/frc/team2412/robot/commands/intake/IntakeTestCommand.java

```diff
@@ -0,0 +1,26 @@
+package frc.team2412.robot.commands.intake;
+
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+import frc.team2412.robot.subsystem.IntakeSubsystem;
+
+public class IntakeTestCommand extends SequentialCommandGroup {
+
+    private final IntakeSubsystem subsystem;
+
+    public IntakeTestCommand(IntakeSubsystem subsystem) {
+        this.subsystem = subsystem;
+        addCommands(
+                // first test
+                new IntakeExtendCommand(subsystem), new WaitCommand(0.5),
```

Comment:
Might consider using the diagnostic command instead of replicating it.

Reply:
The test command is actually different from the diagnostic command; the test command has an extra 3 lines testing if the motors run while the intake is retracted.

Reply:
Correct, up until those three lines they are the same command.  Instead of the five cut and pasted commands from IntakeDiagnosticCommand, can it just run the IntakeDiagnosticCommand?

Reply:
That makes sense, thank you.